curvefs/src/client/inode_wrapper.h

```diff
@@ -159,6 +164,12 @@ class InodeWrapper : public std::enable_shared_from_this<InodeWrapper> {
         return inode_.length();
     }
 
+    // Get inode's length.
+    // REQUIRES: |mtx_| is held
+    uint64_t GetLengthLocked() const {
```

Comment:
It's deleted.

I see this problem, and this is because we don't have a coding style, even though we claim that our code is adhering to google c++ coding style, but is not, we broke many rules.

About this problem, the suffix `Locked` means the caller must hold the lock. This convention is adopted by many famous open source projects, like rocksdb [example](https://github.com/facebook/rocksdb/blob/5d6005c780d47da020d40f2917cfc9ff18de1f85/utilities/transactions/lock/point/point_lock_manager.cc#L473), chromium [example](https://github.com/chromium/chromium/blob/3b188694c0d20e3aafb46ab144b5e396cc0a233a/base/trace_event/trace_log.cc#L569).